__tests__/app.test.js

```diff
@@ -117,7 +117,49 @@ describe('GET /api/articles/:article_id', () => {
         .get('/api/articles/99999999')
         .expect(404)
         .then(({ body }) => {
-            expect(body.msg).toBe('Not found')
+            expect(body.msg).toBe('Article_id not found!')
+        })
+    })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('GET 200 sends an array of comment objects for the passed article_id with correct properties', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comments).toHaveLength(11);
+            body.comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1);
+                expect(typeof comment.comment_id).toBe('number');
+                expect(typeof comment.votes).toBe('number');
+                expect(typeof comment.created_at).toBe('string');
+                expect(typeof comment.author).toBe('string');
+                expect(typeof comment.body).toBe('string');
```

Comment:
These typeof tests do the job, however, have a look at ```.toMatchObject``` and ```expect.any``` which will make these so much nicer 😁